CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+### citus v8.1.0 (December 17, 2018) ###
```

Comment:
Maybe add `* Fixes a bug that could cause maintenance daemon panic` which we've packported to 8.0.2, but merged in 8.1.0 as well. (@velioglu pointed that we should add these type of patches to here as well.)

Reply:
Also, maybe add an entry for https://github.com/citusdata/citus/pull/2496.

`Only allow transmit from pgsql_job_cache directory`